config/base/scene/simple_desktop.json

```diff
@@ -241,6 +235,79 @@
+              "time": "2000-11-01/current/P1D",
+              "layers": "fires_viirs",
+              "preFetch": 0
+            },
+            {
+              "name": "Land Surface Temperature [Day] D",
+              "copyright": "NASA",
+              "url": "https://neo.sci.gsfc.nasa.gov/wms/wms?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&CRS=CRS:84&BBOX=-180,-90,180,90&FORMAT=image/png",
+              "width": 1600,
+              "height": 800,
+              "time": "2000-02-24/2000-04-25/P1D,2000-04-28/2000-08-05/P1D,2000-08-18/2000-12-30/P1D,2001-01-01/2001-06-15/P1D,2001-07-03/2002-03-19/P1D,2002-03-29/2002-04-14/P1D,2002-04-16/2003-12-16/P1D,2003-12-24/2004-12-30/P1D,2005-01-01/2008-12-20/P1D,2008-12-23/2008-12-30/P1D,2009-01-01/2012-12-30/P1D,2013-01-01/2016-02-18/P1D,2016-02-28/2016-12-30/P1D,2017-01-01/2019-11-30/P1D",
+              "layers": "MOD_LSTD_D",
+              "preFetch": 5
+            },
+            {
+              "name": "Land Surface Temperature [Day] M",
+              "copyright": "NASA",
+              "url": "https://neo.sci.gsfc.nasa.gov/wms/wms?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&CRS=CRS:84&BBOX=-180,-90,180,90&FORMAT=image/png",
+              "width": 1600,
+              "height": 800,
+              "time": "2000-02-01/2019-11-01/P1M",
+              "layers": "MOD_LSTD_M",
+              "preFetch": 5
+            },
+            {
+              "name": "angeeichtes Radarkomposit (RW)",
+              "copyright": "NASA",
+              "url": "https://maps.dwd.de/geoserver/ows?service=wms&version=1.3.0&request=GetMap&CRS=EPSG:1000004&BBOX=-180,-90,180,90&FORMAT=image/png",
+              "width": 1600,
+              "height": 800,
+              "time": "2020-01-07T08:30:00.000Z/2020-01-08T08:30:00.000Z/PT10M",
+              "layers": "dwd:RADOLAN-RW",
+              "preFetch": 5
+            },
+            {
+              "name": "Radarvorhersage (Basis RX)",
+              "copyright": "DWD",
+              "url": "https://maps.dwd.de/geoserver/ows?service=wms&version=1.3.0&request=GetMap&CRS=EPSG:1000004&BBOX=-180,-90,180,90&FORMAT=image/png",
+              "width": 1600,
+              "height": 800,
+              "time": "2020-01-08T08:45:00.000Z,2020-01-08T08:50:00.000Z,2020-01-08T08:55:00.000Z,2020-01-08T09:00:00.000Z,2020-01-08T09:05:00.000Z,2020-01-08T09:10:00.000Z,2020-01-08T09:15:00.000Z,2020-01-08T09:20:00.000Z,2020-01-08T09:25:00.000Z,2020-01-08T09:30:00.000Z,2020-01-08T09:35:00.000Z,2020-01-08T09:40:00.000Z,2020-01-08T09:45:00.000Z,2020-01-08T09:50:00.000Z,2020-01-08T09:55:00.000Z,2020-01-08T10:00:00.000Z,2020-01-08T10:05:00.000Z,2020-01-08T10:10:00.000Z,2020-01-08T10:15:00.000Z,2020-01-08T10:20:00.000Z,2020-01-08T10:25:00.000Z,2020-01-08T10:30:00.000Z,2020-01-08T10:35:00.000Z,2020-01-08T10:40:00.000Z,2020-01-08T10:45:00.000Z,2020-01-08T10:50:00.000Z",
+              "layers": "dwd:FX-Produkt"
+            }
+          ]
+        },
+        "Mars": {
```

Comment:
Delete the Mars section here and add to simple-bodies. The reason is that this server is DLR internal and external people do not have access to that one